app/views/catalog/_index_default.html.erb

```diff
@@ -5,13 +5,13 @@
   <% index_fields(document).each do |solr_fname, field| -%>
     <% if should_render_index_field? document, field %>
       <% if document[solr_fname].length == 1 || !document[solr_fname].is_a?(Array) %>
-        <% valdir = getdir(document[solr_fname])%>
+        <% valdir = document[solr_fname].first.dir %>
```

Comment:
This approach won't work for Solr single-valued fields (`!document[solr_fname].is_a?(Array)`) because they are of type String. Can we add an elsif for strings where we call `document[solr_fname].dir` instead?